cli/src/lib/npm/npmLibDefs.js

```diff
@@ -181,8 +181,10 @@ async function extractLibDefsFromNpmPkgDir(
   return libDefs;
 }
 
-async function getCacheNpmLibDefs(cacheExpiry) {
-  await ensureCacheRepo(cacheExpiry);
+async function getCacheNpmLibDefs(cacheExpiry, skipCache) {
```

Comment:
just to be explicit about it
```suggestion
async function getCacheNpmLibDefs(cacheExpiry, skipCache = false) {
```